tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -40,22 +41,7 @@ jobs:
     - template: templates/set-featurizer-build-flag-step.yml
       parameters:
         is_featurizers_build: $(is_featurizers_build)
-
-    - task: CmdLine@2
-      displayName: 'Download azcopy'
-      inputs:
-        script: |
-          curl -so azcopy.tar.gz -L 'https://aka.ms/downloadazcopy-v10-linux'
-          tar -zxvf azcopy.tar.gz --strip 1
-        workingDirectory: $(Build.BinariesDirectory)
-
-    - task: PythonScript@0
-      displayName: 'Download test data'
-      inputs:
-        scriptPath: '$(Build.SourcesDirectory)/tools/ci_build/github/download_test_data.py'
-        arguments: --test_data_url $(TestDataUrl) --build_dir $(Build.BinariesDirectory)
-        pythonInterpreter: '/usr/bin/python3'
-        workingDirectory: $(Build.BinariesDirectory)
+   
```

Comment:
it's nice to not have to download the data anymore. but can we verify the integrity of the data somehow? e.g., verify the checksum of the contents

Reply:
You're concern is valid, tut it may take even longer than downloading from cloud. For example, if the data size is 30GB, and if disk speed is 100MB, then roughly it will need 5 minutes. 
Now I mount these data as readonly, so maybe it's less often to get corrupted. I suggest maybe we can watch it for a while, to see how often data corruption could happen.